pkg/devfile/devfile.go

```diff
@@ -0,0 +1,25 @@
+package devfile
```

Comment:
maybe it's a bit better to rename file to header.go

Reply:
Fixed